docs/api/series.md

```diff
@@ -0,0 +1,113 @@
+<!-- front-matter
+id: api-series
+title: series()
+hide_title: true
+sidebar_label: series()
+-->
+
+# series()
+
+Combines task functions and/or composed operations into larger operations that will be executed one after another, in sequential order. The composed operations from `series()` and `parallel()` can be nested to any depth.
```

Comment:
Perhaps more accurate to say that the framework doesn't impose limitations on depth -- of course the callstack will eventually blow up at some point.